db/v-1/tables/test-execution.xml

```diff
@@ -24,7 +24,7 @@
             <column name="execution_id" type="bigint">
                 <constraints foreignKeyName="fk_test_execution_execution" references="execution(id)" nullable="false"/>
             </column>
-            <column name="status" type="varchar(50)" defaultValue="READY">
+            <column name="status" type="varchar(50)" defaultValue="READY_FOR_TESTING">
```

Comment:
I believe it will break checksum checks for an existing DB, you should use something like `addDefaultValue` in the same changeset, as `update`

Reply:
Ok it seems that we dont't need any liquibase patch here so I will just delete it.

Reply:
misunderstood you. fixing

Reply:
We (with @akuleshov7) found out that READY is not present in real databases and only present in test data so we need to update checksum on all instances and update test-data.

Reply:
Ok, sounds fine to me.